### ISSUE TITLE
Cannot preview variables

### ISSUE THREAD

**Issue by shoryaconsul** [spyder-ide/spyder#9276]
### What steps will reproduce the problem?

Running memory-intense code. When I click on the variable in the variable explorer, I get this error.

### Traceback
```python-traceback
  File "C:\Users\shory\Anaconda3\lib\site-packages\spyder\widgets\variableexplorer\collectionseditor.py", line 476, in createEditor
    if not editor.setup_and_check(value, title=key, readonly=readonly):
  File "C:\Users\shory\Anaconda3\lib\site-packages\spyder\widgets\variableexplorer\arrayeditor.py", line 620, in setup_and_check
    self.data.flags.writeable = True
ValueError: cannot set WRITEABLE flag to True of this array
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.5.6
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```
pyflakes >=0.6.0  :  1.3.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.1.3 (OK)
sphinx >=0.6.6    :  1.4.6 (OK)
rope >=0.9.4      :  0.9.4-1 (OK)
jedi >=0.9.0      :  0.9.0 (OK)
nbconvert >=4.0   :  4.2.0 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.16.3 (OK)
sympy >=0.7.3     :  1.0 (OK)
cython >=0.21     :  0.24.1 (OK)
qtconsole >=4.2.0 :  4.4.3 (OK)
IPython >=4.0     :  5.1.0 (OK)
matplotlib >=2.0.0:  2.2.2 (OK)
pylint >=0.25     :  1.5.4 (OK)
```

**Comment by MaximRosh** [comment]
same problem

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.3.4). Please update by opening the Anaconda Prompt and running there

    conda update spyder
